build/Build.csproj

```diff
@@ -2,12 +2,20 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+	<SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Colors.Net" Version="1.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
     <PackageReference Include="NuGet.Versioning" Version="5.10.0" />
     <PackageReference Include="System.IO.Abstractions" Version="2.1.0.227" />
     <PackageReference Include="WindowsAzure.Storage" Version="9.3.2" />
+	<PackageReference Include="Microsoft.Extensions.DependencyModel" Version="7.0.0" />
   </ItemGroup>
+	<ItemGroup>
+		<None Remove="runtimes.json" />
+	</ItemGroup>
+	<ItemGroup>
+		<EmbeddedResource Include="runtimes.json" />

```

Comment:
Code for reference - [Link](https://github.com/Azure/azure-functions-host/blob/b612457d32902cd4b6b9c5b32b5eecb20240549c/src/WebJobs.Script/WebJobs.Script.csproj#L38)